api/app/models/membership.rb

```diff
@@ -3,4 +3,13 @@ class Membership < ApplicationRecord
   belongs_to :workspace
 
   enum :role, { admin: 0, member: 1 }
+
+  def last_admin?
+    return false unless admin?
```

Comment:
I assume this is kind of a shortcut? Like if user is not an admin he can leave a workspace safely?
Then change the function name to something like `available_to_leave?`, because this line will not tell you if there is last admin